taxi/views.py

```diff
@@ -52,3 +53,43 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class CarCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Car
+    fields = "__all__"
```

Comment:
Using `fields = "__all__"` in a CreateView or UpdateView can be risky as it exposes all fields of the model to the form. Consider specifying only the fields that should be editable by the user to prevent unintended data exposure.